cli/src/deploy/mod.rs

```diff
@@ -622,22 +602,28 @@ pub async fn list_deploy_configs() {
   }
 }
 
-async fn get_creds() -> HashMap<String, CredentialsProfile> {
+async fn get_creds() -> HashMap<String, Credential> {
   let home = std::env::var("HOME").unwrap();
   let file_name = &format!("{}/{}", home, CREDENTIALS_FILE);
   let file = OpenOptions::new().read(true).open(file_name);
   if let Err(_) = file {
+    println!("file error");
```

Comment:
Debug line to remove? Or promote to `error!`?

Reply:
oops shouldn't be there